N1C WAF Security Dashboard Release (#1834)
nginx/documentation: content/nginx-one-console/waf-integration/waf-security-dashboard/dashboard-metrics-reference.md

---
nd-content-type: reference
nd-docs: DOCS-000
nd-product: NONECO
title: Security dashboard reference
description: "Tabs, global controls, and widget-to-dimension mapping for the F5 WAF for NGINX security monitoring dashboard."
weight: 400
toc: true

> nd-keywords: "security dashboard, tabs, global filters, widgets, dimensions, F5 WAF for NGINX"

Reviewer: ```suggestion
nd-keywords: "security dashboard, tabs, global filters, widgets, dimensions, F5 WAF for NGINX, Add Filter, time window, Support ID, violation rating, signature accuracy, signature risk, threat campaign, event logs, Main tab, Advanced tab, Event Logs tab, analytics API, WAF dashboard reference"
```

nd-summary: >
  Use this reference to look up how the security monitoring dashboard is organized and which underlying dimension each widget reads from.
  This article covers the dashboard tabs, the global filter and time controls, and the mapping from widget to API dimension.
  Each widget has an in-product tooltip explaining what its values mean; this article focuses on what is not in those tooltips.
nd-audience: operator
---

## Overview

> Use this reference to look up how the F5 WAF for NGINX security monitoring dashboard is organized and which underlying dimension each widget reads from. Every widget in the dashboard has an in-product tooltip that explains what the displayed values mean. This article mainly covers the dashboard structure, the global controls that affect every widget, and the mapping you need when you want to reproduce a widget's view through the [analytics API]({{< ref "/nginx-one-console/waf-integration/waf-security-dashboard/query-events-api.md" >}}).

Reviewer: ```suggestion
Use this reference to look up how the F5 WAF for NGINX security monitoring dashboard is organized and which underlying dimension each widget reads from. Every widget in the dashboard has an in-product tooltip that explains what the displayed values mean. This article covers the dashboard structure, the global controls that affect every widget, and the mapping you need when you want to reproduce a widget's view through the [analytics API]({{< ref "/nginx-one-console/waf-integration/waf-security-dashboard/query-events-api.md" >}}).
```

| Tab | Purpose |
|:--- |:--- |
| **Main** | High-level summary of WAF activity in the selected window. Shows attack counts, threat intelligence, attack volume over time, and top-N breakdowns by geolocation, policy, IP, violation, and signature. This is the landing tab for triage. |
| **Advanced** | Deeper analytics for tuning and investigation. Use this tab to drill into signature attributes (risk, accuracy, CVE), violation context, and other dimensions that are too detailed for the Main tab. |
| **Event Logs** | A filterable list of individual security events. Open an event to see its support ID, full violation and signature detail, request context, and the raw matched request. This tab is the drill-down target when you want to inspect specific events behind a metric. |

---

## Global controls

Two controls at the top of the page apply to every widget on every tab.

### Time window

Selects the query window for all widgets. The picker offers preset windows from **Last 5 minutes** to **Last 60 days**. You can also select a custom range by highlighting an area of interest on any time-series chart.

> Time-series widgets bucket their data automatically based on the selected window — shorter windows produce finer buckets.

Reviewer: ```suggestion
Time-series widgets bucket their data automatically based on the selected window. Shorter windows produce finer buckets.
```

{{< call-out "note" >}}Security events are retained for **90 days**, but the dashboard time window picker tops out at the **last 60 days**. To query the full retention window, use the [analytics API]({{< ref "/nginx-one-console/waf-integration/waf-security-dashboard/query-events-api.md" >}}), which accepts any time range up to 90 days.{{< /call-out >}}

### Add Filter

Applies one or more filter expressions to every widget on every tab. The dashboard supports the following filters:

| Filter | Description |
|:--- |:--- |

> | **Config Sync Group** | Scope events to instances belonging to a specific Config Sync Group. |

Reviewer: ```suggestion
| **Config Sync Group** | Instances belonging to a specific Config Sync Group. |
```
There is a mix of noun and verb phrases in this table. All entries should follow the same pattern

| **Country** | Two-letter country code derived from the client IP. |

> | **Destination Hostname** | The HTTP `Host` header sent by the client. Use this to scope by the application being attacked. |

Reviewer: ```suggestion
| **Destination Hostname** | The HTTP `Host` header sent by the client. |
```

The removed sentence is usage guidance not a description of the field

Reviewer: we can maybe add another column for usage guidance. the same issue affects another couple of rows in this table

| **Hostname** | The NGINX instance hostname (the data plane host). Use this to scope by the data plane producing the events. |

> | **Instance** | Scope events to a specific NGINX instance by object identity rather than hostname. |

Reviewer: ```suggestion
| **Instance** | A specific NGINX instance, identified by object identity rather than hostname. |
```

| **Violation** | The name of a triggered violation. |
| **Violation Context** | Where in the request the violation occurred: `cookie`, `header`, `parameter`, `request`, or `URI`. |
| **Violation Context Key** | The field name (for example, the parameter or header name) where the violation occurred. |
| **Violation Context Value** | The field value where the violation occurred. |
| **Violation Rating** | The numeric severity rating (0–5) F5 WAF for NGINX assigned to the violation. |

Use filters to scope the dashboard to a specific policy, instance, hostname, country, IP, signature, violation, or any combination. To go from a metric on the dashboard to the underlying events, apply the relevant filter and switch to the **Event Logs** tab.

Every dashboard filter is also available through the analytics API. To call the same filters from the API, see the [API reference guide]({{< ref "/nginx-one-console/api/api-reference-guide.md" >}}).

---

## Main tab widgets

Each widget on the Main tab has a tooltip describing what it displays. The table below adds context the tooltips do not cover, such as how rows are counted and what distinct counts each Top-N table reports.

> | Widget | Notes |

Reviewer: should this table also have a dimension or API field column?

Reviewer 2: It was certainly was getting complicated to do that because it's not really 1:1 mapping. 

It seemed to be easier to reference the query-events-api.md on this page and that would give more details. Also the API docs are very thorough, so it was redundant to repeat much of that here.

| **Top Attack IP Addresses** | Each row shows hits, distinct URIs, violations, and policies for the client IP. |
| **Top Violations** | Each row shows hits, distinct IPs, URIs, and policies for the violation. |
| **Top Signatures** | Each row shows hits, distinct URIs, IPs, violations, and policies for the signature. |
| **Top Subviolations** | Each row shows hits, distinct IPs, URIs, and policies for the sub-violation. |
| **Top Attack URIs** | Each row shows hits, distinct IPs, violations, and policies for the URI. |
| **Request Methods** | Donut chart showing the share of events for each HTTP method observed in the window. |
| **Response Codes** | Donut chart showing the share of events for each response code observed in the window. |

To reproduce these widgets through the analytics API, see [Query security events through the API]({{< ref "/nginx-one-console/waf-integration/waf-security-dashboard/query-events-api.md" >}}).

---

## Advanced tab widgets

The Advanced tab exposes signature- and violation-level analytics for tuning and deeper investigation. Widgets on this tab read from the same data store as the Main tab and respect the same global filters and time window.

> | Widget | Notes |

Reviewer: should this table also have a dimension or API field column?

Reviewer 2: same reason: https://github.com/nginx/documentation/pull/1834#discussion_r3059082056

| **Advanced** | Deeper analytics for tuning and investigation. Use this tab to drill into signature attributes (risk, accuracy, CVE), violation context, and other dimensions that are too detailed for the Main tab. |
| **Event Logs** | A filterable list of individual security events. Open an event to see its support ID, full violation and signature detail, request context, and the raw matched request. This tab is the drill-down target when you want to inspect specific events behind a metric. |

---

## Global controls

Two controls at the top of the page apply to every widget on every tab.

### Time window

Selects the query window for all widgets. The picker offers preset windows from **Last 5 minutes** to **Last 14 days**. You can also select a custom range by highlighting an area of interest on any time-series chart.

Time-series widgets bucket their data based on the selected window — shorter windows produce finer buckets.

| Selected window | Default bucket size for time-series widgets |
|:--- |:--- |
| Last 5–30 minutes | 1 minute |
| Last 1–6 hours | 1 minute |
| Last 12–24 hours | 1 hour |
| Last 2–7 days | 1 hour |
| Last 14 days | 6 hours |

> 

Reviewer 3: Changing this to have `Last 30 days` and `Last 60 days` instead of `Last 14 days`. Ref https://gitlab.com/f5/nginx/one/saas/control-plane/-/merge_requests/6076#b6ff2499a841a5c57f8a6fd70d27f2a6a1b60cef

Reviewer 2: ok will consider.

Reviewer 2: using "5 minutes to 60 days". Thanks.

| **Top Signature CVEs** | Each row shows hits, distinct URIs, IPs, violations, and policies for signatures that reference the CVE. |
| **Top Threat Campaigns** | Each row shows hits, distinct URIs, IPs, violations, and policies for the threat campaign. |
| **Top Attacked Instances** | Each row shows hits, distinct URIs, IPs, violations, and policies, scoped to the NGINX instance hostname (not the HTTP `Host` header). |

To reproduce these widgets through the analytics API, see [Query security events through the API]({{< ref "/nginx-one-console/waf-integration/waf-security-dashboard/query-events-api.md" >}}).

---

## Event Logs tab

The Event Logs tab lists individual security events matching the global filters and time window. The tab has two parts:

- A **Security Events** time-series chart at the top showing event volume by request status (`blocked` and `alerted`), the same view as the **Attack Requests Over Time** widget on the Main tab.
- A paginated **events table** below the chart, where each row corresponds to one security event.

### Events table columns

| Column | Description |
|:--- |:--- |

> | **Status** | The final WAF decision: `blocked` or `alerted`. |

Reviewer 3: This table also accepts `passed` as a Status but we only allow filtering for `blocked` or `alerted`, if relevant

Reviewer 2: added
